default.nix

```diff
@@ -99,7 +99,12 @@ in {
     exactDeps = false;
   }; in {
     inherit hsPkgs;
-    ormoluShell = shellFor (ps: [ ps.ormolu ]);
+    ormoluShell = (shellFor (ps: [ ps.ormolu ]))
```

Comment:
It bother me to modify the state of the system implicitly like this, but I will leave the decision to @amesgen.

Reply:
I suggested an alternative that does not have this caveat: https://github.com/tweag/ormolu/pull/974#pullrequestreview-1262270371